dev/ProgressBar/ProgressBarWinUI_themeresources.xaml

```diff
@@ -0,0 +1,7 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
```

Comment:
Rename it to ProgressBar_themeresources.xaml?

Reply:
Cannot use that name as it clashes with ProgressBar_themeresources.xaml in CommonStyles. But since it's not needed, I removed this file for now.

@jevansaks I am not sure how the hand-off is going to work between the framework version and this, do you have more info on this.